monix-execution/jvm/src/main/scala/monix/execution/tracing/CorrelationId.scala

```diff
@@ -0,0 +1,16 @@
+package monix.execution.tracing
+
+import monix.execution.misc.{TracingContext, TracingContextCompanion}
+
+/**
+  * A tracing context that allows to propagate a correlation id
+  * through an execution.
```

Comment:
From the ScalaDoc I don't understand the purpose of this.

If you can give a sample, please do so, or you can mention something like "*Used in X*" or something like that.